transcendence-app/src/shared/relationship/relationship.module.ts

```diff
@@ -0,0 +1,10 @@
+import { Module } from '@nestjs/common';
+import { RelationshipService } from './relationship.service';
+import { DbModule } from '../db/db.module';
+
+@Module({
+  imports: [DbModule],
+  providers: [RelationshipService],
+  exports: [RelationshipService],
+})
+export class RelationshipModule {}
```

Comment:
Why do we need a new module for this? Why this can't be a part of a user? What else do we expect to be blocked in the future? What does "relationship" means?